modules/hal_nordic/nrfx/nrfx_glue.h

```diff
@@ -251,6 +251,45 @@ void nrfx_busy_wait(uint32_t usec_to_wait);
 
 //------------------------------------------------------------------------------
 
+/**
+ * @brief Macro for writing back cache lines associated with the specified buffer.
+ *
+ * @param[in] p_buffer Pointer to the buffer.
+ * @param[in] size     Size of the buffer.
+ */
+#define NRFY_CACHE_WB(p_buffer, size) \
+    do {                              \
+        (void)p_buffer;               \
+        (void)size;                   \
+    } while (0)
```

Comment:
I'm not sure if compiler won't complain about unused variables

Reply:
It's a macro, so these are not variables.

Reply:
but it might be called in a function where `p_buffer` is used only in the macro. Wouldn't that cause a compilation error is such a case? I'll remove it and we'll see